roles/mongodb/tasks/main.yml

```diff
@@ -1,25 +1,8 @@
 ---
-- name: Add mongodb key
-  become: yes
-  apt_key:
-    keyserver: keyserver.ubuntu.com
-    id: 42F3E95A2C4F08279C4960ADD68FA50FEA312927
-    state: present
-  tags: [databases, mongodb]
+- name: Install mongodb rhel
+  include: mongodb_rhel.yaml
+  when: ansible_distribution == 'CentOS'
```

Comment:
We won't need  `deb` and `rpm` here, because of https://github.com/StackStorm/ansible-st2/pull/65#discussion_r94613855.